### PR TITLE
Update rooms-release-note.md

### PR DESCRIPTION
**Pull request by Yoojin-Jung**
Draft: https://microsoft-my.sharepoint-df.com/:w:/p/yoojinjung/ETuWZtGLZGlMuNcsk1u0FLUBgH6cpyDKuY93oXzGhHs3bQ?e=xoer88

Release date: near the end of 3/22/2023

### CODE REVIEW

**Comment by bbriggs00** [comment]
May AllFather damn your souls with torment of spirit day in and out

File: Teams/rooms/rooms-release-note.md [hunk 1]
```diff
@@ -95,6 +96,18 @@ Teams Rooms is governed by the Modern Lifecycle Policy. For more information, se
 
 ## Microsoft Teams Rooms feature introduction and issue resolution
 
+### 4.16.40.0 (3/22/2023)
+
+Introduced in this update:
+
+- A revitalized Teams Rooms User interface [Learn more](../rooms/mtr-home-refresh.md#enable-refreshed-home-screen-design)
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-release-note.md:103]:
```suggestion
- A revitalized Teams Rooms User interface. [Learn more](../rooms/mtr-home-refresh.md#enable-refreshed-home-screen-design)
```

File: Teams/rooms/rooms-release-note.md [hunk 2]
```diff
@@ -95,6 +96,18 @@ Teams Rooms is governed by the Modern Lifecycle Policy. For more information, se
...
+- Enhanced Layout Switcher <sup>1</sup>
+- Support Teams premium meetings protection policies (Watermark, End-to-end Encryption for meetings, and Sensitivity labels) <sup>2</sup>
+
+ <sup>1</sup> Functionality in the layout switcher is contingent upon the type of Teams Rooms license.
+
+ <sup>2</sup> Meeting organizer should have a valid license to use these policies. Learn more at [Microsoft Teams Premium licensing - Microsoft Teams | Microsoft Learn](../teams-add-on-licensing/licensing-enhance-teams.md)
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-release-note.md:109]:
```suggestion
 <sup>2</sup> Meeting organizers should have a valid license to use these policies. Learn more at [Microsoft Teams Premium licensing](../teams-add-on-licensing/licensing-enhance-teams.md)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Yoojin-Jung, kindly commit the suggested changes. Many thanks.

**Pull request closed without merging**